tests/wdio/secondaryNavigation-spec.js

```diff
@@ -18,3 +18,41 @@ Terra.describeViewports('secondary scroll multiple directories', ['huge'], () =>
     Terra.validates.element({ selector: '#root' });
   });
 });
+
+Terra.describeViewports('secondary nav scroll', ['huge'], () => {
+  describe('should not scroll when item is visible', () => {
+    before(() => browser.url('/secondary-nav-test/terra-dev-site/secondary-scroll-test/filler/filler-test'));
+    Terra.it.validatesElement('Before', { selector: '#root' });
+    it('selects filler 04 Test', () => {
+      browser.click('[class*="Extension-module__extension"]');
+      browser.keys('Filler 04');
+      browser.waitForVisible('[class*="list-item"]');
+      browser.click('[class*="list-item"]');
+    });
+    Terra.it.validatesElement('After', { selector: '#root' });
+  });
```

Comment:
I think it's cleaner to contain your test to a single it block.
```suggestion
  it('should not scroll when item is visible', () => {
    browser.url('/secondary-nav-test/terra-dev-site/secondary-scroll-test/filler/filler-test'));
    Terra.validatesElement('Before', { selector: '#root' });
    browser.click('[class*="Extension-module__extension"]');
    browser.keys('Filler 04');
    browser.waitForVisible('[class*="list-item"]');
    browser.click('[class*="list-item"]');
    Terra.validatesElement('After', { selector: '#root' });
  });
```

Reply:
Updated here https://github.com/cerner/terra-dev-site/pull/261/commits/f7cc5a3c33dc0374e91fefb452b76a895a8e7543.